butterfree/configs/db/cassandra_config.py

```diff
@@ -150,6 +156,28 @@ def stream_checkpoint_path(self, value: str) -> None:
             "STREAM_CHECKPOINT_PATH"
         )
 
+    @property
+    def read_consistency_level(self) -> Optional[str]:
+        """Read consistency level for Spark."""
+        return self.__read_consistency_level
+
+    @read_consistency_level.setter
+    def read_consistency_level(self, value: str) -> None:
+        self.__read_consistency_level = value or environment.get_variable(
+            "CASSANDRA_READ_CONSISTENCY_LEVEL", "LOCAL_ONE"
+        )
```

Comment:
It is a bit odd to image a call with `value=None` and have this function to try to read from an env var. I would expect a regular assignment, at most an error/warning if trying to set an invalid value such as `None` (is it invalid?).

Reply:
I sorta followed already existing setters like 
https://github.com/quintoandar/butterfree/blob/b4b7628a32e172cf5d631d2d89674f45848bdd59/butterfree/configs/db/cassandra_config.py#L155
Where if value = None (nothing is passed) then it defaults to the env var. The difference in this one is that even if no env var was set, I kept the sprk default

Reply:
That's right @fvcaputo :wink: